src/devices/bus/hp_hil/hlebase.cpp

```diff
@@ -0,0 +1,152 @@
+// license:BSD-3-Cl ause
+// copyright-holders:Sergey Svishchev
+#include "emu.h"
+#include "hlebase.h"
+#include "hlekbd.h"
```

Comment:
I think this is a leftover from copy/pasting - the base shouldn't depend on the implementations.